src/plugins.js

```diff
@@ -29,8 +29,6 @@ var middleware;
 	Plugins.lessFiles = [];
 	Plugins.clientScripts = [];
 	Plugins.acpScripts = [];
-	Plugins.customLanguages = {};
-	Plugins.customLanguageFallbacks = {};
```

Comment:
What happens to fallbacks now? If a plugin defines `en-GB/foobar.json`, what does accessing `/assets/language/ru/foobar.json` return?